src/pages/db/_components/Hero.astro

```diff
@@ -6,7 +6,7 @@ import Pill from "./Pill.astro"
 ---
 
 <section
-	class="align-center landing-section relative flex h-[calc(100dvh-5rem)] max-h-[40rem] content-center items-center overflow-hidden bg-[#6628C6] bg-cover bg-no-repeat px-4 py-16 md:px-8"
+	class="align-center landing-section relative z-[11] flex h-[calc(100dvh-5rem)] max-h-[40rem] content-center items-center overflow-hidden bg-[#6628C6] bg-cover bg-no-repeat px-4 py-16 md:px-8"
```

Comment:
Why 11 specifically? 😅 super specific z-index numbers kind of scare me, let's stick to default TW values if possible

Reply:
Ha, this is a weird one. 11 is above 10 but below 20. 😅

I guess I can adjust the other layers on the page